docs/docs/networks/sepolia.mdx

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 3
+id: sepolia
+---
+
+# Sepolia
+
+Sepolia is a Proof-of-Stake(PoS) testnet where dApp developers can test their applications, and previously, was
+Proof-of-Work(PoW). What differentiates Sepolia from other Ethereum testnets is that most of its PoW validators
+are public validators. Combined with Sepolia being a testnet, this creates the chance that validators will stop
+validating and the chain will fail to finalize, making Sepolia one of the best testnets to see how decentralized
+applications will perform under the harshest conditions.
```

Comment:
This is referring to the time Sepolia was PoW and you talking using present tense. Update this please and focus on Sepolia being a public network but with permissioned validator sets.